app/src/main/java/org/simple/clinic/home/overdue/compose/NoPendingPatients.kt

```diff
@@ -21,6 +21,7 @@ fun NoPendingPatients(modifier: Modifier = Modifier) {
   Card(
       modifier = modifier
           .fillMaxWidth()
+          .padding(8.dp)
```

Comment:
We can use `dimenResource` for this, since we already have spacing tags defined

Reply:
Done